tests/stests/workloads/create_workloads.robot

```diff
@@ -80,3 +80,34 @@ Test Ankaios Podman create a container with custom name
     Then the JSON array "${dict_array}" shall contain key "Labels" with subkey "name" which matches the expression "^nginx.\\w+.agent_A$"
     Then the JSON array "${dict_array}" shall contain array "Names" which contains value "test_workload1"
     [Teardown]    Clean up Ankaios
+
+# [stest->swdd~podman-kube-create-workload-apply-manifest~1]
+# [stest->swdd~podman-kube-create-workload-creates-config-volume~1]
+# [stest->swdd~podman-kube-create-workload-creates-pods-volume~1]
+# [stest->swdd~podman-kube-delete-workload-downs-manifest-file~1]
+# [stest->swdd~podman-kube-delete-removes-volumes~1]
+
+Test Ankaios Podman create kube workload
+    [Setup]    Run Keywords    Setup Ankaios
+
+    # Preconditions
+    # This test assumes that all pods and volume in the podman have been created with this test -> clean it up first
+    Given Podman has deleted all existing pods
+    Given Podman has deleted all existing volumes
+    And Ankaios server is started with "ank-server --startup-config ${CONFIGS_DIR}/kube.yaml"
+    And Ankaios agent is started with "ank-agent --name agent_A"
+    And all workloads of agent "agent_A" have an initial execution state
+    When user triggers "ank get workloads"
+    Then the workload "nginx" shall have the execution state "Running" on agent "agent_A"
+    # Check config and pods volumes has been created
+    When user executes system app "podman volume ls --format=json"
+    ${dict_array}=    And the result is valid JSON
+    Then the JSON array "${dict_array}" shall contain key "Name" which matches the expression "^nginx.\\w+.agent_A.(config|pods)$"
+    # Check config and pods volumes are deleted when workload is deleted
+    When user triggers "ank delete workload nginx"
+    And user triggers "ank get workloads"
+    Then the workload "nginx" shall not exist
+    When user executes system app "podman volume ls --format=json"
+    ${dict_array}=    And the result is valid JSON
+    Then the JSON array "${dict_array}" shall contain key "Name" which not matches the expression "^nginx.\\w+.agent_A.(config|pods)$"
```

Comment:
Here I am not sure, normally if there is no volume anymore it returns empty list. So must it be rephrased to "the JSON array shall not contain the key "Name"...".

In addition, just a question: is this done for easier testing that two tests are mixed into one? Normally, testing the remove of the volumes and that a workload is created are separate tests. But for me it would be ok if it is for simplicity here.

Reply:
The test was changed, pls take a lot at the new impl.